tests/test_factory.py

```diff
@@ -304,8 +304,8 @@ def test_validate_fails():
     content["this key is not in the schema"] = ""
 
     expected = (
-        "Additional properties are not allowed "
-        "('this key is not in the schema' was unexpected)"
+        "Additional properties are not allowed ('this key is not in the schema' was"
```

Comment:
Unrelated and less readable

Reply:
I'll try to revert these formatting changes.